indexd/guid/blueprint.py

```diff
@@ -0,0 +1,60 @@
+import flask
+import uuid
+
+blueprint = flask.Blueprint("guid", __name__)
+
+
+@blueprint.route("/guid/mint", methods=["GET"])
+def mint_guid():
+    """
+    Mint a GUID that is valid for this instance of indexd. The intention
+    of this endpoint is to allow generating valid GUIDs to be indexed
+    WITHOUT actually creating a new record yet.
+
+    Allows for a `count` query parameter to get bulk GUIDs up to some limit
+    """
+    count = flask.request.args.get("count", 1)
+    max_count = 10000
+
+    try:
+        count = int(count)
+    except Exception:
+        return f"Count {count} is not a valid integer", 400
+
+    # clip value on 0, max_count
+    if count < 0:
+        count = 0
+    elif count > max_count:
+        count = max_count
```

Comment:
maybe we should return an error for these 2 cases? or at least when count > max_count instead, so that users expecting a specific number of GUIDs don't run into unexpected errors?